src/Basque/Mews.Fiscalizations.Basque/DtoToModelConverter.cs

```diff
@@ -14,6 +14,23 @@ internal static SendInvoiceResponse Convert(
         string tbaiIdentifier,
         String1To100 signatureValue)
     {
+        if (response is null)
+        {
+            var errorDescription = "Server error. Please try again.";
+            return new SendInvoiceResponse(
+                xmlRequestContent: xmlRequestContent,
+                xmlResponseContent: xmlResponseContent,
+                qrCodeUri: qrCodeUri,
+                tbaiIdentifier: tbaiIdentifier,
+                received: DateTime.UtcNow,
+                state: InvoiceState.Refused,
```

Comment:
I don't think its correct for us to decide whether the invoice is refused or not, this should be only taken from the response.

also if the invoice is refused, i don't think we should be returning the qr code